sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2649,3 +2652,84 @@ case class Sentences(
     copy(str = newFirst, language = newSecond, country = newThird)
 
 }
+
+case class ToPrettyString(expr: Expression, timeZoneId: Option[String] = None)
+  extends UnaryExpression with ImplicitCastInputTypes with TimeZoneAwareExpression{
+  import ToPrettyString._
+
+  require(children.nonEmpty, s"$prettyName() should take at least 1 argument")
```

Comment:
what does it do? This is a `UnaryExpression`...

Reply:
Removed..